CHANGELOG.markdown

```diff
@@ -1,3 +1,8 @@
+0.6.2
+-----
+
+* Use automatic flags
```

Comment:
A little more detail here would be appreciated. In particular:

1. What was `transformers-compat` doing before, and why was this a problem?
2. It's also worth noting here that this eliminates the need for the concurrent version approach.

Reply:
This still needs doing.